Classes/Private/Manager/Database/OCTDBManager.m

```diff
@@ -257,6 +257,27 @@ - (RLMResults *)callsWithChat:(OCTDBChat *)chat
     return results;
 }
 
+- (OCTDBCall *)getOrCreateCallWithFriendNumber:(NSInteger)friendNumber
+{
+    OCTDBChat *chat = [self getOrCreateChatWithFriendNumber:friendNumber];
+    __block OCTDBCall *call = [[self callsWithChat:chat] lastObject];
```

Comment:
I think to be consistent it would be better leave everything as it is now. We'll see, maybe we'll refactor it in future.